fdbserver/BackupWorker.actor.cpp

```diff
@@ -153,6 +153,7 @@ struct BackupData {
 			    .detail("Version", savedVersion);
 			return;
 		}
+		ASSERT_WE_THINK(backupEpoch == oldest);
```

Comment:
@jzhou77 It turns out the ASSERTION failed. I'm confused why would backupEpoch < oldest?

Reply:
`backupEpoch` is the epoch the worker is working on. Say current epoch is 6. The `backupEpoch` can be 4. The `recruitedEpoch` is 6.

The master doesn't have a fix for oldest backup epoch. So on master, the oldest is always initialized as current epoch, e.g., 6. So the assertion fails here for workers working on Epoch 4. If you apply the [fix](https://github.com/apple/foundationdb/pull/2642/commits/19f6394dc99746b822d9e7418636dc52837a1794), I think the assertion won't fail.

Reply:
I see. Since your PR#2642 is almost finished, I just comment out the ASSERT in this PR and would you mind adding this ASSERT into your PR#2642?

Reply:
Will do.